### ISSUE TITLE
Wrong syntax for MySQL composite primary key

### ISSUE THREAD

**Issue by AlessioDP** [drizzle-team/drizzle-kit-mirror#119]
drizzle-kit is generating a wrong syntax SQL code on create table with multiple primary keys.

By following the documentation, we can create a simple table with two primary keys like this:
```typescript
export const usersToGuilds = mysqlTable('users_to_guilds', {
    userId: int('user_id'),
    guildId: int('guild_id')
}, (table) => ({
    pk: primaryKey(table.userId, table.guildId)
}));
```

The generated SQL code is:
```sql
CREATE TABLE `users_to_guilds` (
	`user_id` int NOT NULL,
	`guild_id` int NOT NULL,
	CONSTRAINT `users_to_guilds_user_id_guild_id` PRIMARY KEY("user_id","guild_id")
);
```

The part `PRIMARY KEY("user_id","guild_id")` should use back ticks instead of double quote for table columns:
```sql
PRIMARY KEY(`user_id`,`guild_id`)
```

---

Furthermore, the documentation says that the result query should be:
```sql
CREATE TABLE `users_to_guilds` (
	`user_id` int NOT NULL,
	`guild_id` int NOT NULL,
        PRIMARY KEY(`user_id`,`guild_id`)
);
```
The CONSTRAINT part should be added into the doc (?)

---

Using `drizzle-orm@0.27.2` and `drizzle-kit@0.19.4` with `mysql2` driver.

**Comment by AndriiSherman** [comment]
Fixing that

**Comment by AndriiSherman** [comment]
Fixed in `drizzle-kit@0.19.5`, thanks!
